index.html

```diff
@@ -105,6 +107,45 @@ <h3>Floor</h3>
 				<div class="floors"></div> <!-- floors -->
 			</div> <!-- dropdown -->
 		</div> <!-- search-menu -->
+			
+		<div class = "legend-bar">
+			
+			<ul>
+				<h1>Display Legend</h1>
+				<li>	
+						<div class="switchmenucontainer">
```

Comment:
Should follow the project's css class name convention of kabob case. ie `switch-menu-container`